packages/visualizations/src/components/Chart/types.ts

```diff
@@ -39,6 +39,8 @@ export interface ChartOptions {
     description?: string;
     /** Link button to source */
     source?: Source;
+    /** Cutout percentage for doughnut charts */
+    cutout?: string;
```

Comment:
Maybe we can move this option in a sub object. IMO, it should not be on the top level of options as this is specific to pie and doughnut charts. 

We can handle this later when implementing other new charts.